api/config/rolepermission.php

```diff
@@ -76,6 +76,7 @@
         'teamMembers' => 'teamMembers',
         'role' => 'role',
         'directiveForm' => 'directiveForm',
+        'poolAssessments' => 'poolAssessments',
```

Comment:
Hmm, I'm not sure we need a new permission for this.  I think I'd be fine with just using the same permissions as pool and treaing them as part of the pool.

Reply:
Yeah, the issue mentioned add new permissions but I wasn't sure what was actually needed. Cluttering with redundant permissions was also something I was concerned about. 

How does this look, leveraging a role check instead.

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/07cc2e58-b390-406f-9a58-600b6d1ea09a)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/7166e9e2-67e4-4a04-a49d-2ec3ddf481e4)

Reply:
Well, hope you liked the above https://github.com/GCTC-NTGC/gc-digital-talent/pull/8022/commits/5f9ecf3e22372ae6515205cedf55c4f3bc8c0aad

Reply:
Nice.  :+1:

Reply:
I'm all for re-using existing permissions if it makes sense, but we shouldn't check Roles directly, to be consistent.

Looking at our existing permissions... If we _really_ want to use an existing permission, we might use the `view-team-pool`. I think anyone who is currently allowed to view draft pools would also be allowed to view the assessment plan.

I'm personally more worried about using permissions in unintuitive ways than I am of creating lots of them. And I think that soon enough we'll need some new permissions for recording assessments, anyway.